portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['Delicious Food', 'Pet Tortoises', '你好，谷歌！', 'Computer & Actuarial', 'Travel', 'Disney Movies'];
```

Comment:
These don't look like greetings.

Reply:
Thank you for the advice! I'm sorry that I kept the original name for the function and thus make it confusing. 
The function is actually used as a random "my keyword" generator in the website. And I have changed all names and descriptions for the function from "greeting" to "keyword" to try to resolve the confusion in the latest commit.
```function addRandomKeyword() {
           const keywords =
                ['Delicious Food', 'Pet Tortoises', '你好，谷歌！', 'Computer & Actuarial', 'Travel', 'Disney Movies'];
```